browser/ChangeLog.md

```diff
@@ -0,0 +1,17 @@
+Note: This is the change log file for Azure Storage JavaScript Client Library.
+
+2017.02 Version 2.0.0-prerelease.1
```

Comment:
* Generated browser compatible JavaScript files based on Microsoft Azure Storage SDK for Node.js 2.0.0